README.md

```diff
@@ -1,141 +1,112 @@
+| ``snmp_group.context.access_list`` | dict | Configures access list entries for the group with context. |
+| ``snmp_group.context.access_list.access`` | string | Configures access-list associated with the group. |
+| ``snmp_group.context.access_list.ipv6`` | string | Configures IPv6 access-list associated with the group. |
+| ``snmp_group.context.view`` | dict | Configues view entries for the group with context. |
+| ``snmp_group.context.view.notify`` | string | Configures notify view associated with the group. |
+| ``snmp_group.context.view.read`` | string | Configures read view associated with the group. |
+| ``snmp_group.context.view.write`` | string | Configures write view associated with the group. |
+| ``snmp_group.context.state`` | choice(absent, present) | If set to absent, deletes the context entries with the group. |
+| ``snmp_group.state`` | string, choice(absent, present*) | If set to absent, deletes the associated SNMP group. |
+
+
+> **NOTE**: Asterisk (*) denotes the default value if none is specified. 
+
+## Connection variables
+Ansible Dell EMC Networking roles require the following connection information to establish communication with the nodes in your inventory. This information can exist in the Ansible group_vars or host_vars directories, or in the playbook itself.
+
+| Key | Required | Choices | Description |
+| -- | -- | -- | -- |
+| ``host`` | yes | | Hostname or address for connecting to the remote device over the specified ``transport`` — value of this key is the destination address for the transport. |
+| ``port`` | no | | Port used to build the connection to the remote device — if the value of this key does not specify the value, the value defaults to 22. |
+| ``username`` | no | | Configures the username that authenticates the connection to the remote device. The value of this key authenticates the CLI login. If this key does not specify a value, the value of ``ANSIBLE_NET_USERNAME`` is used instead. |
+| ``password`` | no | | Specifies the password that authenticates the connection to the remote device. If this key does not specify the value, the value of ``ANSIBLE_NET_PASSWORD`` is used instead. |
+| ``authorize`` | no | yes, no*  | Instructs the module to enter Privileged mode on the remote device before sending commands. If this key does not specify the value, the value of ``ANSIBLE_NET_AUTHORIZE`` is used instead. If not specified, the device attempts to execute all commands in non-Privileged mode.|
+| ``auth_pass`` | no | | Specifies the password to use if required to enter Privileged mode on the remote device. If ``authorize`` is set to no, this key is not applicable. If this key does not specify the value, the value of ``ANSIBLE_NET_AUTH_PASS`` is used instead. |
+| ``transport`` | yes  | cli* | Configures the transport connection to use when connecting to the remote device. This key supports connectivity to the device over CLI (SSH).  |
```

Comment:
Can you please remove the transport variable, this is removed in the latest Ansible.